lib/iris/fileformats/netcdf.py

```diff
@@ -899,14 +919,18 @@ def write(self, cube, local_keys=None, unlimited_dimensions=None,
             http://www.unidata.ucar.edu/software/netcdf/docs/BestPractices.html#bp_Packed-Data-Values
             If this argument is a type (or type string), appropriate values of
             scale_factor and add_offset will be automatically calculated based
-            on `cube.data` and possible masking. For masked data, `fill_value`
-            is taken from netCDF4.default_fillvals. For more control, pass a
-            dict with one or more of the following keys: `dtype` (required),
-            `scale_factor`, `add_offset`, and `fill_value`. Note that automatic
-            calculation of packing parameters will trigger loading of lazy
-            data; set them manually using a dict to avoid this. The default is
-            `None`, in which case the datatype is determined from the cube and
-            no packing will occur.
+            on `cube.data` and possible masking. For more control, pass a dict
+            with one or more of the following keys: `dtype` (required),
+            `scale_factor` and `add_offset`. Note that automatic calculation of
```

Comment:
Isn't this the reference you were looking form, saying that  automatic packing realises the data ?